client/app/src/js/controllers/screenshots.js

```diff
@@ -13,18 +13,22 @@ function ScreenshotsController(screenshots, Screenshot) {
         console.log(data);
```

Comment:
"this.screenshots" is set to "screenshots" that we inject directly to the controller.